DESCRIPTION

```diff
@@ -25,10 +25,10 @@ URL: https://github.com/bcgov/bcmaps,
     http://bcgov.github.io/bcmaps/
 BugReports: https://github.com/bcgov/bcmaps/issues
 Depends:
-    sf (>= 0.9),
+    sf (>= 1.0),
     R (>= 2.10)
 Imports: 
-    bcdata (>= 0.3.0),
+    bcdata (>= 0.4.1),
```

Comment:
This one, no particular reason but I also don't think it hurts. The spatial ones have all had their internal plumbing rewired for the removal of rgdal/rgeos, so although I haven't dug into each one, I think it's good for them to updated to avoid any potential nasty surprises

Reply:
Yeah sure, a safe bet.